### PR TITLE
Refactor vagrant environment settings

### PR DESCRIPTION
**Pull request by malukenho**

### CODE REVIEW

File: ansible/playbook.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    - name: Install packages
+      apt: package={{item}} state=present
+      with_items:
+          - curl
+          - git
+          - php5-fpm
+          - php5-cli
+          - php5-curl
+          - php5-mongo
+          - php5-sqlite
```
Comment (abdala) [inline ansible/playbook.yml:28]: why we need this?

File: ansible/playbook.yml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+          - openjdk-7-jre
```
Comment (abdala) [inline ansible/playbook.yml:29]: Java? Really? Is it for ElasticSearch? What are your ideias?

File: ansible/playbook.yml (hunk 1 above)
Comment (malukenho) [inline ansible/playbook.yml:28]: It's necessary to use broadway (Logs I guess)

File: ansible/playbook.yml (hunk 2 above)
Comment (malukenho) [inline ansible/playbook.yml:29]: Doing some experiments, not really necessary for now.

File: ansible/playbook.yml (hunk 1 above)
Comment (abdala) [inline ansible/playbook.yml:28]: :+1:

**Pull request merged**